views/xibo-audience-connector-form-settings.twig

```diff
@@ -34,6 +34,10 @@
         {% set title %}{% trans "API Key" %}{% endset %}
         {% set helpText %}{% trans "Enter your API Key from Xibo." %}{% endset %}
         {{ forms.input("apiKey", title, interface.getSetting("apiKey"), helpText) }}
+
+        {% set title %}{% trans "Batch Size" %}{% endset %}
```

Comment:
This is inside the check for apiKey being a provider setting? I think it needs its own condition.